lib/CXGN/Trial/ParseUpload/Plugin/MultipleTrialDesignExcelFormat.pm

```diff
@@ -587,18 +584,6 @@ sub _validate_with_plugin {
       $self->_set_parse_warnings(\%warnings);
   }
 
-  ## PLOT POSITION OVERALL VALIDATION
-  foreach my $tk (keys %seen_plot_keys) {
-      foreach my $pk (keys %{$seen_plot_keys{$tk}} ) {
-          my $plots = $seen_plot_keys{$tk}{$pk};
-          my $count = scalar(@{$plots});
-          if ( $count > 1 ) {
-              my @pos = split('-', $pk);
-              push @error_messages, "More than 1 plot is assigned to the position row=" . $pos[0] . " col=" . $pos[1] . " trial=" . $tk . " plots=" . join(',', @$plots);
```

Comment:
Should this be removed? Just checking

Reply:
That one is awkward - it is not connected with any .xlsx functionality. Looks more like some problem with rebasing / branches. I will check why it is even there.